src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -67,10 +67,13 @@
     private final String authHeader;
 
     private final String baseApiPath;
+    
+    private final int timeout;
 
-    public JiraRestService(URI uri, JiraRestClient jiraRestClient, String username, String password) {
+    public JiraRestService(URI uri, JiraRestClient jiraRestClient, String username, String password, int timeout) {
```

Comment:
To retain the binary compatibility you would need to keep the original constructor and to mark it as deprecated